src/utils.ts

```diff
@@ -30,6 +30,18 @@ import {
 import { Mjolnir } from "./Mjolnir";
 import config from "./config";
 import { ClientRequest, IncomingMessage } from "http";
+import { default as parseDuration } from "parse-duration";
+
+// Define a few aliases to simplify parsing durations.
+
+parseDuration["days"] = parseDuration["day"];
+parseDuration["weeks"] = parseDuration["week"] = parseDuration["wk"];
+parseDuration["months"] = parseDuration["month"];
+parseDuration["years"] = parseDuration["year"];
```

Comment:
What is actually going on here?

Reply:
By default, with `parseDuration`, if I'm not mistaken, you can write `2 day` but not `2 days`.

Reply:
oh ok, i see